eeraster.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import pandas as pd
+import ee
+import eeddf
+import math
+import raster
+from typing import Tuple, List
+from multiprocessing import Pool
+
+
+_PARALLEL_OPS = 30
+_CHUNK_SIZE = 80
+_LONGITUDE_COLUMN_NAME = "long"
+_LATITUDE_COLUMN_NAME = "lat"
+_demXfab = None
+_dem = None
```

Comment:
I have often seen config options created as an object with defaults, and I personally like when libraries do that so I can change things as needed. Otherwise, I have had to resort to reflection and similar techniques to modify such things, though that is less of an issue here than with some technologies (e.g. WPF, JavaFX) where the user is not expected to modify source code in their environment. Just something to think about.

Reply:
I've added a method to set options which makes it discoverable.  For now, the method affects global settings, which I think is most likely to be useful, but at a future time we can experiment with a per-call option object.